src/app/components/authDialog/authDialog.component.ts

```diff
@@ -8,16 +8,13 @@ import { AuthService } from '../../services/api/auth.service';
   styleUrls: ['./authDialog.component.scss'],
 })
 export class AuthDialogComponent {
-  username = '';
-  password = '';
-
   constructor(
     private dialogRef: MatDialogRef<AuthDialogComponent>,
     private apiService: AuthService,
```

Comment:
Do you use this service in this component?

Reply:
Nice catch